changelogs/unreleased/gh-3211-per-module-log_level.md

```diff
@@ -0,0 +1,8 @@
+## feature/box
+
+* Added new `print_module_name` option to `log.cfg{}` and `box.cfg{}` that
+  enables printing in the log the name of a module from which the logging
+  function was called (gh-3211).
+
+* Supported per-module log level setting via `log.cfg{level_modules = {...}}`
+  or `box.cfg{log_level_modules = {...}}` (gh-3211).
```

Comment:
I guess we need to come to terms. We have different objects on `log.new` and `require('log')` and both of them are configurable thru `log_level_modules`. I guess we can't name the result of `log.new` a log module. 

May be we should name both objects loggers in this aspect and rename option to `log_level_logger`. And we can say that the result of `require('log')' is logger with module name.

Reply:
Ok, let's call the result of both `require('log')` and `log.new()` — a logger.
«modules» in `log_level_modules` refer to the names of the loggers.